docker/readme.md

```diff
@@ -43,13 +43,18 @@ docker run --publish 10931:10933 --tty --interactive --env "ListeningPort=10931"
 ### Environment variables
 
 - **ServerApiKey**: The API Key of the Octopus Server the Tentacle should register with.
-- **ServerUsername**: If not using an api key, the user to use when registering the Tentacle with the Octopus Servr.
-- **ServerPassword**: If not using an api key, the password to use when registering the Tentacle
+- **ServerUsername**: If not using an API key, the user to use when registering the Tentacle with the Octopus Servr.
+- **ServerPassword**: If not using an API key, the password to use when registering the Tentacle
 - **ServerUrl**: The Url of the Octopus Server the Tentacle should register with.
+- **Space**: The name of the space which the Tentacle will be added to. Defaults to the default space.
 - **TargetEnvironment**: Comma delimited list of environments to add this target to.
 - **TargetRole**: Comma delimited list of roles to add to this target.
-- **TargetWorkerPool**: Comma delimited list of worker pools to add to this target to (not to be used with environments or role variable).
+- **TargetWorkerPool**: Comma delimited list of worker pools to add this target to (not to be used with environment, tenant, tenant tag or role variable).
 - **TargetName**: Optional Target name, defaults to host.
+- **TargetTenant**: Comma delimited list of tenants to add to this target.
+- **TargetTenantTag**: Comma delimited list of tenant tags to add to this target.
+- **TargetTenantedDeploymentParticipation**: The tenanted deployment mode of the target. Allowed values are Untenanted, TenantedOrUntenanted and Tenanted. Defaults to Untenanted.
+- **MachinePolicy**: The name of the machine policy that will apply to this Tentacle. Defaults to the default machine policy.
```

Comment:
From the existing configuration, `MachinePolicy` and `Space` already exist for both images, and `MachinePolicy` exists for Linux. I've added policy to the Windows image, and I've added documentation for these two parameters. Unless there's a reason why we don't want to expose this configuration for some reason?